src/main/java/org/generationcp/ibpworkbench/germplasm/SaveGermplasmListAction.java

```diff
@@ -1,193 +0,0 @@
-/*******************************************************************************
```

Comment:
@gelli-leafnode not clear how this class is related to the Germplasm Details? 
Is it ok to delete this one?

Reply:
Hi @clarysabel, 

This was only used in SaveGermplasmListDialog.java (germplasm) and SaveGermplasmListDialog.